### PR TITLE
sync subviews with master

### PR DESCRIPTION
**Pull request by lynxlynxlynx**
nothing that dramatic in conflicts

### CODE REVIEW

**Comment by lynxlynxlynx** [comment]
wait, I forgot something.

**Comment by lynxlynxlynx** [comment]
ok, it runs now. GCW fails, but that looks recent, related to the latest control creation changes.
Label.SetAnimation aborts; on a newly created Label. It's a nice python object, but the view lookup fails in the bindings.

**Comment by bradallred** [comment]
what is "GCW"?

**Comment by lynxlynxlynx** [comment]
guicommonwindows

File: gemrb/GUIScripts/GUIOPT.py [hunk 1]
```diff
@@ -545,7 +545,8 @@ def LoadGamePress ():
 		LoadMsgWindow.Unload ()
 	LoadMsgWindow = None
 	GemRB.QuitGame ()
-	OpenOptionsWindow()
+	GemRB.SetVar ("SelectedWindow", 0)
```
Comment (bradallred) [inline gemrb/GUIScripts/GUIOPT.py:548]: I don't think `SelectedWindow` is used for anything anymore

**Comment by lynxlynxlynx** [comment]
On Thu, 08 Jun 2017 14:07:30 -0700
Brad Allred <notifications@github.com> wrote:

> bradallred commented on this pull request.
> 
> 
> 
> > @@ -545,7 +545,8 @@ def LoadGamePress ():  
>  		LoadMsgWindow.Unload ()
>  	LoadMsgWindow = None
>  	GemRB.QuitGame ()
> -	OpenOptionsWindow()
> +	GemRB.SetVar ("SelectedWindow", 0)
> 
> I don't think `SelectedWindow` is used for anything anymore
It wasn't at that point, but it is again with the sync with master.

**Pull request merged**